build-release-tools/lib/PlatformClients.py

```diff
@@ -150,6 +152,21 @@ def get_box_versions(self):
         else:
             raise Exception("Failed to get box versions {0}\n{1}".format(self.box, resp.text))
         return versions
+    def get_box_versions_between_upload_time_range(self, begin_time, end_time):
```

Comment:
A little suggestion, add an assertion to make sure the input `end_time > begin_time`. Because sometimes user may input the time in wrong order,  this will help us to identify problem.